src/openlifu/db/volume.py

```diff
@@ -0,0 +1,73 @@
+import json
+from dataclasses import dataclass
+from pathlib import Path
+from typing import Dict, Optional
+
+from openlifu.util.json import PYFUSEncoder
+
+
+@dataclass
+class Volume:
```

Comment:
I wasn't thinking that a new class would be needed for this, but I see that it is pretty nice to have a class. Still, I think it should not be called Volume because it actually just references a volume file rather than being an actual volume. Maybe `VolumeInfo`?

If we do add this class then I think in this PR we should unit test all the functionality. So `to_json`, etc. should all be tested. The suit of tests could be in a `test_volume.py` file

But we could also get by without having this class at all. All that `write_volume` needs to do is write a json file from a dict with `id`, `name`, and `filename`, and this can be done directly in the `Database` class. And then `Database.load_volume_info` (or `Database.get_volume_info`) would just have to read the json file and return that as a dict, as well as returning the filepath to the volume file.